decidim-meetings/lib/decidim/meetings/component.rb

```diff
@@ -28,11 +28,17 @@
   end
 
   component.register_stat :meetings_count,
-                          primary: true,
                           admin: false,
                           priority: Decidim::StatsRegistry::HIGH_PRIORITY,
                           icon_name: "map-pin-line",
                           tooltip_key: "meetings_count_tooltip" do |components, start_at, end_at|
+    meetings = Decidim::Meetings::FilteredMeetings.for(components, start_at, end_at).published.not_withdrawn
+    meetings.count
+  end
+
+  component.register_stat :admin_meetings_count,
+                          primary: true,
+                          admin: false do |components, start_at, end_at|
     meetings = Decidim::Meetings::FilteredMeetings.for(components, start_at, end_at).not_withdrawn
```

Comment:
I actually found a bug: 
1. Visit the admin area, create a meeting from admin, do no publish it, and everything is ok 
2. Edit component settings, and allow Participants to create
3. Visit frontent interface, create a meeting 
4. Withdraw the meeting 
5. Visit admin, and compare the counter and the actual displayed meetings

```suggestion
    meetings = Decidim::Meetings::FilteredMeetings.for(components, start_at, end_at)
```